test/documentation/test_preprocess_documentation.t

```diff
@@ -0,0 +1,134 @@
+Testing interaction between `gospel pps` and Ocaml compiler. The issue is that
```

Comment:
The tests could also use an inner nested comment, since they should be now covered.